internal/pkg/selector/selector.go

```diff
@@ -88,6 +89,8 @@ func (m *Menu) Display() string {
 		} else if keyCode == down {
 			m.CursorPos = (m.CursorPos + 1) % len(m.MenuItems)
 			m.renderMenuItems(true)
+		} else if keyCode == ctrl_c {
+			return "Exited !"
```

Comment:
Return a blank string, and add checks wherever this function is called.

Reply:
ok